radis/params.py

```diff
@@ -65,7 +65,7 @@
 
 See Also
 --------
-:py:func:`~radis.lbl.broadening._check_accuarcy`
+:py:meth:`~radis.lbl.broadening.BroadenFactory._check_accuracy`
```

Comment:
@erwanp this doesn't seem to work. `_check_accuarcy` is a method of `BroadenFactory` class. So I wonder why its not working?

Reply:
Maybe because it's a private method? Try renaming it to check_accuracy (without underscore)

Reply:
It's not very important and we can merge anyway! :)

Reply:
It didn't work. Maybe because of it being a private method. Sure lets merge it :)